### PR TITLE
fix: create Emoji API uses "name" parameter to specify emoji

### PR DESCRIPTION
**Pull request by zaquestion**
Ref: https://docs.gitlab.com/ce/api/award_emoji.html#award-a-new-emoji

### CODE REVIEW

File: award_emojis.go [hunk 1]
```diff
@@ -56,38 +56,47 @@ const (
...
+type ListAwardEmojiOptions ListOptions
+
+// CreateAwardEmojiOptions represents the available options for awarding emoji
+// for a resource
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/award_emoji.html#award-a-new-emoji
+type CreateAwardEmojiOptions struct {
+	Name string `json:"name"`
+}
```
Comment (svanharmelen) [inline award_emojis.go]: Looks good! But please place this declaration above the first `Create...` method.

Comment (zaquestion) [reply award_emojis.go]: Updated, PTAL

**Comment by svanharmelen** [comment]
LGTM! Thanks @zaquestion!

**Comment by svanharmelen** [comment]
Tagged as v0.10.6

**Pull request merged**